lib/tasks/auth_tokens.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+namespace :auth_tokens do
+  desc "Verify each authorized AuthToken that it's still authorized"
+  task :reverify_authorized, %i[count start] => :environment do |_task, args|
+    exit if ENV["READ_ONLY"].present?
+    args.with_defaults(count: 500, start: nil)
+    last_id = "none"
+
+    begin
+      AuthToken
+        .authorized
+        .in_batches(of: args.count, start: args.start)
+        .each do |token_batch|
+          token_batch.each do |token|
+            result = token.still_authorized?
+            fresh_login = token.github_client.user[:login]
```

Comment:
This is going to throw some errors if the token is not authorized anymore. You can see some of the rescues happening in the [still_authorized?](https://github.com/librariesio/libraries.io/blob/bd03a18a2fd6a11292d0096a47613ea362e0c19f/app/models/auth_token.rb#L46) method. If we want the login, we should only fetch it if `result == true`.